ee/tabby-webserver/src/service/repository/mod.rs

```diff
@@ -105,15 +105,15 @@ impl RepositoryService for RepositoryServiceImpl {
     }
 
     async fn resolve_repository(&self, kind: &RepositoryKind, id: &ID) -> Result<Repository> {
-        if let RepositoryKind::Git = kind {
+        if let RepositoryKind::GitConfig = kind {
             if let Ok(index) = config_id_to_index(id) {
```

Comment:
if it's an GitConfig but not able to run `config_id_to_index` successfully, we shall bail instead?